pilot/helpers/AgentConvo.py

```diff
@@ -68,7 +68,7 @@ def send_message(self, prompt_path=None, prompt_data=None, function_calls: Funct
                                                   function_calls=function_calls, prompt_data=prompt_data,
                                                   temperature=self.temperature)
         except TokenLimitError as e:
-            save_development_step(self.agent.project, prompt_path, prompt_data, self.messages, '', str(e))
+            save_development_step(self.agent.project, prompt_path, prompt_data, self.messages, {"text": ""}, str(e))
```

Comment:
This is the only place in the codebase where we're saving `llm_response` as a string instead of a dict.

After this crash, if the person tries to reload the project, they get:

```
TypeError: string indices must be integers, not 'str':
pilot/helpers/agents/Developer.py:132
pilot/helpers/agents/Developer.py:92
pilot/helpers/Project.py:204
```

Exact code number varies by version but it's always here:

https://github.com/Pythagora-io/gpt-pilot/blob/e31c67667dfad32a3c83a6425b86b122b488245e/pilot/helpers/agents/Developer.py#L132